doc/sphinx/proof-engine/tactics.rst

```diff
@@ -2008,7 +2008,7 @@ analysis on inductive or co-inductive objects (see :ref:`inductive-definitions`)
 
    .. coqtop:: reset all
 
-      Lemma le_minus : forall n:nat, n < 1 -> n = 0.
+      Lemma lt_1_r : forall n:nat, n < 1 -> n = 0.
```

Comment:
Why the name change?  There is an lt_1_r theorem with a different statement in NOrder.v--were you trying to reference that?

Theorem lt_1_r : forall n, n < 1 <-> n == 0.

Reply:
> Why the name change?

«Standard» `le_minus` is `∀ n m : nat, n - m <= n` isn’t it?  There’s already another `le_minus` in this section matching that.

> were you trying to reference that?

Yes.

Reply:
The statement of `Nat.lt_1_r` (derived from the theorem in `NOrder.v` that Jim mentioned) is `forall n : nat, n < 1 <-> n = 0`, so here the only difference is `->` vs `<->`.

> «Standard» `le_minus` is `∀ n m : nat, n - m <= n` isn’t it?

It is indeed.